runtime/build.gradle.kts

```diff
@@ -4,6 +4,16 @@ plugins {
     kotlin("multiplatform")
     kotlin("plugin.serialization")
     `maven-publish`
+    // Fetch/sync remote proto files prior to code gen.
+    id("com.tinder.gitquery")
+}
+
+val protoDir = "src/commonMain/proto"
```

Comment:
I didn't gitignore these as including them will help users of pbandk not have to also get protobuf-java just for the includes.